Essentials/src/messages_de.properties

```diff
@@ -272,7 +272,8 @@ maxHomes=§4Du kannst nicht mehr als§c {0} §4Heimaten setzen.
 maxMoney=§4Diese Überweisung überschreitet das Limit des Empfängers.
 mayNotJail=§4Du kannst diese Person nicht einsperren.
 mayNotJailOffline=§4Du darfst abgemeldete Spieler nicht einsperren.
-me=Du
+meSender=ich
```

Comment:
Yea, was just showing how this could solve the original problem. Should i undo b57631c0bd04b61703f1a5a26ab0946b54cdfd8d or will it just be overwritten when merged?

Reply:
It will be overwritten when the PR gets merged and the translations are updated from Crowdin.